packages/js/product-editor/src/blocks/generic/toggle/block.json

```diff
@@ -12,6 +12,9 @@
 			"type": "string",
 			"__experimentalRole": "content"
 		},
+		"help": {
+			"type": "string"
+		},
```

Comment:
One of the criteria's was to hide the `help` text when toggled on:

> Note that we don't want this help text to appear when the toggle is on. So when toggled on, it should still look like this:

Should we provide help text for when it's toggled on or off?

Reply:
if we want to make it scalable, probably we'll have to add another attribute, like `helpOff` or something like that

Reply:
Yeah I did prefer the more scalable option.
Maybe we could follow the format of the `checkedValue`/`uncheckedValue` and use `checkedHelpText` & `uncheckedHelpText`? or just `checkedHelp` would work to, the `text` addition might be slightly more clear.